src/components/cylc/table/deltas.js

```diff
@@ -136,8 +136,10 @@ function applyDeltasAdded (added, table, lookup) {
           const latestJob = latestJobSubmitNum < job.submitNum
             ? lookup[job.id]
             : existingEntry.latestJob
+          // FIXME: Couldn't we avoid setting table[existing.id].latestJob multiple times here?
+          //        If a task has 100 jobs, in the worst case we will call Vue.set 100 times?
```

Comment:
From the [docs](https://v2.vuejs.org/v2/api/#Vue-set), `Vue.set()` is only needed to add a property to a reactive object. But if the properties `jobs` and `latestJob` already exist, there should be no need to use `Vue.set()`, just assign as normal?